torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -1840,6 +1840,29 @@ def _(func, types, args, kwargs):
             weight_tensor = weight_tensor.dequantize()
         return torch.nn.functional.linear(input_tensor, weight_tensor, bias)
 
+@implements(torch.nn.functional.embedding)
+def _(func, types, args, kwargs):
+    # new_arg1 = args[1].dequantize()
+    idx = args[0]
+    int_data, scale, zero_point = args[1].tensor_impl.get_plain()
```

Comment:
should we have an assert on the type of quantization here (block_size)?

Reply:
i think we're ok for now but yeah eventually